proto/iotics/api/feed.proto

```diff
@@ -44,11 +44,13 @@ service FeedAPI {
 }
 
 // A feed representation.
-message Feed {
-  // Feed identifier (unique within the scope of a twin identifier's feed set)
-  FeedID id = 1;
-  // Twin unique identifier (twin to which the feed belongs)
-  TwinID twinId = 2;
+message FeedID {
+  // Feed identifier string representation (simple string) (unique within the scope of a twin identifier's feed set)
+  string id = 1;
+  // Twin identifier string representation (simple string) (twin to which the feed belongs)
+  string twinId = 2;
+  // Host identifier string representation (simple string) (Host to which the twin belongs)
+  string hostId = 3;
 }
 
 // CreateFeedRequestCreate is used to create a new feed in a given twin.
```

Comment:
!!! The CreateFeedRequest Payload needs to be updated.
```
message Payload {
    // ID of the feed to create
    string id = 1;
  }
```
Else data will be duplicated (same for input)